api/contract.go

```diff
@@ -109,6 +109,19 @@ func (r *putFilterOutputRequest) Valid() error {
 	return nil
 }
 
+type AddOptionResponse struct {
+	Option    string     `json:"option"`
+	Self      model.Link `json:"self"`
+	Filter    model.Link `json:"filter"`
+	Dimension model.Link `json:"Dimension"`
+}
+
+// getDimensionOptionsResponse is the response body for GET /filters/{id}/dimensions/{name}/options
+type GetDimensionOptionsResponse struct {
```

Comment:
Could we name this GetFilterDimensionOptionsResponse so it's distinct from e.g. GetFilterOutputDimensionOptionsResponse etc